### DISCUSSION TITLE
Where is the document of the command ‘pnpm view * ‘

### DISCUSSION THREAD

**Discussion by Niefee** [pnpm/pnpm#5556] [Q&A] [answered]
Where is the document of the command ‘pnpm view * ‘，I cannot find the document of the `view` command on the cli command page

**Comment by lvqq** [comment, marked as answer]
It has the same behavior of `npm view *`, we don't have a documentation about it.

**Reply by Niefee** [reply]
Is pnpm compatible with other npm commands? For example `npm info *`

**Reply by lvqq** [reply]
Yes, for these commands we use npm to run. https://github.com/pnpm/pnpm/blob/main/packages/pnpm/src/pnpm.ts#L28~#L60

**Reply by Niefee** [reply]
thank you
